### ISSUE TITLE
ability to suppress the SDK Warning

### ISSUE THREAD

**Issue by voshond** [jangxx/node-vjoy#5]
Hi jang!

i've been using your package alot recently.
However i'm having the issue that Joystick Gremlin doesn't support the new fork of njx3, as i am using this vjoy device as an input.
I already opened a ticket for Joystick Gremlin, but seeing their backlog i don't think it will be implemented very soon.

Do you think we could add an option to disable the warning?

**Comment by jangxx** [comment]
I could move the check into the vJoyDevice constructor, so it would only show up when a new device is initialized (instead of when the library is `require`'d), and we could then have a `disable_version_check` parameter, which would suppress the warning.

But then again, is the warning really _that_ annoying? If you are running different versions of the library and the driver, you are factually running an unsupported setup and I don't see why a little console message warning you of that fact could be that bad.

So basically: Yes, I could move some stuff around and add an option, but I'm not really convinced that it would add much to the library.

**Comment by jangxx** [comment]
Okay, I've now though of another way to disable the warning which is less invasive than what I thought up before. You simply have to set the environment variable `DISABLE_VJOY_VERSION_CHECK` to something, and you won't see the warning anymore. This feature is in version 1.2.2 which I've already published to npm and I think we can close this issue now.
